src/overlays/ovl_i4/fox_sz.c

```diff
@@ -120,132 +129,142 @@ void SectorZ_Missile_Update(Actor* this) {
             break;
     }
 
-    sp2C = gBosses[0].obj.pos.x - this->obj.pos.x;
-
-    if (0) {} //! FAKE
-
-    sp24 = gBosses[0].obj.pos.z - this->obj.pos.z;
-
-    SIN_DEG(gGameFrameCount); // Leftover from a test or commented out code?
-
-    if (sp2C) {} //! FAKE
-    if (sp24) {} //! FAKE
+    /* Leftover from a test or commented out code? */
+    /* =========================================== */
+    x = gBosses[0].obj.pos.x - this->obj.pos.x;
+    y = gBosses[0].obj.pos.y - this->obj.pos.y; // Optimized out?
+    if (1) {} //! FAKE
+    z = gBosses[0].obj.pos.z - this->obj.pos.z;
+    SIN_DEG(gGameFrameCount);
+    if (x && z) {} //! FAKE
+    /* =========================================== */
 
     if (this->aiType < 100) {
-        if (sp24) {}                                    //! FAKE
-        sp2C = fabsf(this->fwork[4] - this->obj.pos.x); //! FAKE
-        sp20 = SIN_DEG((this->index * 45) + gGameFrameCount) * 5000.0f;
-        sp1C = COS_DEG((this->index * 45) + (gGameFrameCount * 2)) * 5000.0f;
+        if (z) {} //! FAKE
+        z = fabsf(this->fwork[6] - this->obj.pos.z); //! FAKE
+
+        // Escorts wooble movement
+        xPitch = SIN_DEG((this->index * 45) + gGameFrameCount) * 5000.0f;
+        yPitch = COS_DEG((this->index * 45) + (gGameFrameCount * 2)) * 5000.0f;
```

Comment:
This is them flying around the missile during the first missile cutscene.

Reply:
Yes it is. Are you suggesting there's something innacurate in the comment?